frontend/models.py

```diff
@@ -1,5 +1,153 @@
 from __future__ import unicode_literals
 
+import os
+import re
+import subprocess
+import requests
+import json
+
+from filelock import Timeout, FileLock
+
+from django import forms
 from django.db import models
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+from generic_confirmation import signals
+from generic_confirmation.forms import DeferredForm
+
+
+class PageEdit(models.Model):
+    page = models.CharField(max_length=100, null=True)
+    patch = models.CharField(max_length=5000, null=True)
+    email = models.EmailField(null=True)
+    comment = models.CharField(max_length=150, null=True)
+    date = models.DateTimeField(null=True)
```

Comment:
It's a good practice in django to add `blank=True` everytime you do a `null=True` because some super shitty db ask for that.